test/server-unit/db.spec.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../../server/lib/db');
+const { expect } = require('chai');
+var _ = require('lodash');
+
+var dbTest = function () {
+
+  before(function () {
+    var params = {
+      host     : 'localhost',
+      user     : 'bhima',
+      password : 'HISCongo2013',
+      database : 'bhima_test',
+    };
+    db.init(params);
+
+  });
+
+  it('Should check the connection to mysql', function (done) {
+    var fx = db.pool.getConnection(function (err, result) {
+      if (err) {
+        done(err);
+        return;
+      }
+      expect(true).to.equal(true);
+      done();
+    });
+
+  });
+
+  it('#Should try to retrieve data from a specific table (unit)', () => {
+    return db.exec('SELECT * FROM unit', {})
+      .then((rows) => {
+				const test = rows.length > 0;
+        expect(test).to.equal(true);
```

Comment:
One of the advantages of chai is that it makes common operations like this easy - using [lengthOf](http://chaijs.com/api/bdd/#method_lengthof) you can chain this together.  Like this:
```js
.then(rows => expect(rows).to.have.lengthOf(2));
```

As a rule, tests should be as specific as possible.  That way, you know each function is performing exactly the way you expect it to at all times.  If any change happens, a test will break and you will have to change either the test or fix the function.